From: Ingo Saitz <ingo@debian.org>
Date: Sat, 29 Oct 2011 14:46:34 +0200
Subject: dlopen libesd.so.0 rather than libesd.so

Bug #163938: Change the library name dlopen uses to libesd.so.0, since we hard
depend on libesd0-dev. Probably this should be integrated into autoconf?
---
 drivers/drv_esd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/drv_esd.c b/drivers/drv_esd.c
index 4fa13fe..5fee86c 100644
--- a/drivers/drv_esd.c
+++ b/drivers/drv_esd.c
@@ -100,7 +100,7 @@ static BOOL ESD_Link(void)
 	if (libesd) return 0;
 
 	/* load libesd.so */
-	libesd=dlopen("libesd.so",RTLD_LAZY|RTLD_GLOBAL);
+	libesd=dlopen("libesd.so.0",RTLD_LAZY|RTLD_GLOBAL);
 	if (!libesd) return 1;
 
 	/* resolve function references */
